src/init.cpp

```diff
@@ -411,7 +410,7 @@ bool AppInit2(int argc, char* argv[])
     }
 
     // Check and update minium version protocol after a given time.
-    if (time(NULL) >= ENABLE_PHASE_TWO_PRIMES)
+    if (GetTime() >= ENABLE_PHASE_TWO_PRIMES)
```

Comment:
Partly unsigned/signed related as time(NULL) is signed but mainly we should be using the built in GetTime method instead of time(NULL) anyway.